packages/core/tsconfig.json

```diff
@@ -2,6 +2,7 @@
   "extends": "../../tsconfig.json",
   "include": ["src"],
   "compilerOptions": {
-    "noImplicitAny": false
+    "noImplicitAny": false,
+    "experimentalDecorators": true
```

Comment:
Could you provide some more reasoning about why we should add this?

Reply:
Mostly it's just related to being able to have static classes typechecked by TypeScript (until I find what should be the "real" answer) than using `// @ts-ignore` in the meantime.

On this note, any suggestions for typechecking a static class? If there's nothing, I'm also happy to ditch the whole `FeatureFlags.{methods}` idea away although it'll impact backwards-compatibility

Reply:
May sound silly, but how about making a regular class with regular methods, and export an instance of that class? Just don't go with statics at all.

Reply:
Oh oops I totally misread your comment. Hmm actually that might be a good idea!

Reply:
^ that :p

Reply:
Done :)